modules/nf-scil/image/resample/main.nf

```diff
@@ -0,0 +1,59 @@
+process IMAGE_RESAMPLE {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_1.6.0.sif':
+        'scilus/scilus:1.6.0' }"
+
+    input:
+    tuple val(meta), path(image)
+
+    output:
+    tuple val(meta), path("*_resampled.nii.gz"), emit: image
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def voxel_size = task.ext.voxel_size ? "--voxel_size " + task.ext.voxel_size : ""
+    def interp = task.ext.interp ? "--interp " + task.ext.interp : ""
+    def volume_size = task.ext.volume_size ? "--volume_size " + task.ext.volume_size : ""
+    def ref = task.ext.ref ? "--ref" + task.ext.ref : ""
+    def iso_min = task.ext.iso_min ? "--iso_min" + task.ext.iso_min : ""
+    def enforce_dimensions = task.ext.enforce_dimensions ? "--enforce_dimensions" + task.ext.enforce_dimensions : ""
+    def f = task.ext.f ? "--f" + task.ext.f : ""
```

Comment:
You are not using those parameters in the script.

Reply:
I included all the function's parameters, regardless of usage. Should we limit to parameters passed in the script?

Reply:
Yes, future dev will include them when they become wanted